src/index.ts

```diff
@@ -42,7 +42,9 @@ function install(Vue: App, initConf: VueGtmUseOptions = { id: "" }): void {
   }
 
   // Add to vue prototype and also from globals
-  Vue.config.globalProperties.$gtm = new GtmPlugin(pluginConfig.id);
+  const gtmInstance = new GtmPlugin(pluginConfig.id);
+  Vue.config.globalProperties.$gtm = gtmInstance;
+  Vue.provide("gtmInstance", gtmInstance);
```

Comment:
About the `"gtmInstance"` naming, is there any convention or other plugins that also using `xInstance`?
Otherwise, what was your idea behind the `Instance` suffix? Is there a better name like `GTM`, `gtmPlugin`, `gtmPluginInstance`, ...?

Also we should maybe add a section to the README so others can also benefit from it in their `setup` methods?